app/auth/sign-up/signupApi.tsx

```diff
@@ -0,0 +1,61 @@
+const URL = "http://159.223.203.135:3000/api/auth/signup";
```

Comment:
Should we change to localhost:3000?
This doesn't connect to the backend and the database.

Reply:
This connects to a live DB but I can change it to localhost if you prefer
![image](https://github.com/SeattleColleges/nsc-events-nextjs/assets/35015441/f687f9d0-1ece-43a2-adf7-80bc8572b280)